lighthouse-core/test/audits/legacy-javascript-test.js

```diff
@@ -144,11 +144,12 @@ describe('LegacyJavaScript audit', () => {
       'String.prototype[\'repeat\'] = function() {}',
       'Object.defineProperty(String.prototype, "repeat", function() {})',
       'Object.defineProperty(String.prototype, \'repeat\', function() {})',
-      'Object.defineProperty(window, \'WeakMap\', function() {})',
+      // Currently are no polyfills that declare a class. May be in the future.
```

Comment:
I'm hopeful that we can somehow get `preset-modules` to workaround the bugs that require Promises to be polyfilled. keeping this around + the code in legacy-javascript that detects this.